ps_featuredproducts.php

```diff
@@ -72,6 +72,8 @@ public function install()
             && $this->registerHook('deleteproduct')
             && $this->registerHook('categoryUpdate')
             && $this->registerHook('displayHome')
+            && $this->registerHook('displayOrderConfirmation2')
```

Comment:
wow, what an ugly hook :P is there any chance to use some widget in this place instead?